library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -75,8 +75,8 @@ class FsSnapshot
       "/usr/bin/snapper --no-dbus --root=%{root} --csvout list-configs " \
       "--columns config,subvolume | /usr/bin/grep \"^root,\" >/dev/null".freeze
 
-    CREATE_SNAPSHOT_CMD = "/usr/lib/snapper/installation-helper --step 5 --root-prefix=%{root} " \
-    "--snapshot-type %{snapshot_type} --description %{description}".freeze
+    CREATE_SNAPSHOT_CMD = "/usr/bin/snapper --no-dbus --root=%{root} create "\
+      "--type %{snapshot_type} --description %{description}".freeze
```

Comment:
does snapper support such change? ( I mean from step 5 to create ) ?

Reply:
Yes, actually it is the preferred way.